core/src/apps/cardano/__init__.py

```diff
@@ -6,6 +6,12 @@
 CURVE = "ed25519"
 SEED_NAMESPACE = [HARDENED | 44, HARDENED | 1815]
 
+BYRON_PURPOSE = 44 | HARDENED
```

Comment:
I wonder whether this shouldn't be an anum. Same below

Reply:
Enums are not supported. I could use a class, but from the first PR it seems, that Trezor guys don't like classes. They proposed a dict. I suppose something like:
```
PURPOSES = {
    "byron": 44 | HARDENED,
    "shelley": 1852 | HARDENED
}
```
What do you think?